Source/WebGPU/WGSL/TypeCheck.cpp

```diff
@@ -289,11 +306,25 @@ void TypeChecker::inferred(Type* type)
     m_inferredType = type;
 }
 
-void TypeChecker::unify(Type* lhs, Type* rhs)
+bool TypeChecker::unify(Type* lhs, Type* rhs)
 {
-    // FIXME: Implement all the rules and report a type error otherwise
+    if (shouldDumpInferredTypes)
+        dataLogLn("[unify] '", *lhs, "' <", RawPointer(lhs), ">  and '", *rhs, "' <", RawPointer(rhs), ">");
```

Comment:
WTFLogAlways() :(

Reply:
[I replied in another PR](https://github.com/WebKit/WebKit/pull/8830#discussion_r1082238809) asking whether you felt strongly about using `WTFLogAlways` over `dataLog`, as I don't see why one is strictly better than other, but I don't think you replied. I searched for when to use one over the other, and as far as I could tell it seemed to be a preference thing. `dataLog` does seem more flexible than `WTFLogAlways`, so I'd personally prefer to stick to that if you are not strictly opposed.